ocean_lib/assets/v4/asset.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import copy
+import json
+import os
+from pathlib import Path
+from typing import Optional
+
+from enforce_typing import enforce_types
+
+from ocean_lib.ocean.util import get_web3
+from ocean_lib.services.v4.service import NFTService
+
+
+@enforce_types
+class V4Asset:
+    """Asset class to create, import, export, validate Asset/DDO objects for V4."""
+
+    def __init__(
```

Comment:
I wanted to create a named tuple that includes all these required parameters and just pass that named tuple instead of this list. What do you think @calina-c, @DMats?

Reply:
If the v3 asset has arguments, I'd rather keep these as arguments too. Anyone can use the `*args` syntax if they want to